src/App.js

```diff
@@ -10,8 +12,10 @@ import {
 function App() {
   return (
     <div>
-      <Navbar />
-
+      <Navbar/>
+      <Carrousel/>
+      <CardsContainer/>
+      <Footer/>
```

Comment:
<Carrousel/> and <CardsContainer/> are already included in the  HomePage page, please remove these two elements here and move <Footer/> below the switch component